libgalois/include/katana/PropertyGraph.h

```diff
@@ -395,11 +409,11 @@ class KATANA_EXPORT PropertyGraph {
       const std::vector<std::string>& node_properties,
       const std::vector<std::string>& edge_properties) const;
 
-  /// Construct node & edge TypeSetIDs from node & edge properties
-  /// Also constructs metadata to convert between types and TypeSetIDs
-  /// Assumes all boolean or uint8 properties are types
+  /// Construct node & edge EntityTypeIDs from node & edge properties
+  /// Also constructs metadata to convert between atomic types and EntityTypeIDs
+  /// Assumes all boolean or uint8 properties are atomic types
   /// TODO(roshan) move this to be a part of Make()
```

Comment:
I think this TODO is fairly urgent, BTW. Is there a reason not to just add the call and make this private?

Reply:
I know. It needs to be moved to the partitioner. It's not as simple as moving it into the constructor.

Reply:
Is there a JIRA issue for this? I think it's worth tracking.